src/components/Calculator.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import '../App.css';
+
+class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="calculator-container">
+        <div className="row">
+          <div className="display">
+            <div className="input">0</div>
+          </div>
+        </div>
+        <div className="row">
+          <div className="operation">AC</div>
+          <div className="operation">+/-</div>
+          <div className="operation">%</div>
+          <div className="operation right">/</div>
```

Comment:
[OPTIONAL]

- I would highly recommend using this `÷` sign instead of `/` to follow the standard.